D2.Detours.patches/1.10f/D2Common.patch.cpp

```diff
@@ -274,9 +274,9 @@ static PatchAction patchActions[GetOrdinalCount()] = {
     PatchAction::Ignore,                         /*C*/ //   D2COMMON_10238_Return                                               @10238
     PatchAction::FunctionReplacePatchByOriginal,       //   D2COMMON_10239_Return0                                              @10239
     PatchAction::FunctionReplaceOriginalByPatch, /*C*/ //   INVENTORY_AllocInventory                                            @10240
-    PatchAction::FunctionReplacePatchByOriginal,       //   INVENTORY_FreeInventory                                             @10241
```

Comment:
Did you have time to test all those functions ?
Should I give those a try ?

Reply:
I tested everything in game that's marked with C now and didn't notice anything weird, but you can definitely also give it a try.